docs/benchmarks/data-serving.md

```diff
@@ -25,6 +25,8 @@ $ docker run --name cassandra-server --privileged --net host cloudsuite/data-ser
 ```
 ### Multiple Server Containers
 
+Please do not start these server containers on the same machine, because every of them will listen on the same port 7199 and thus fail. See [issue #329](https://github.com/parsa-epfl/cloudsuite/issues/329) for more details. 
```

Comment:
This line should be replaced with the following sentence:
Please note the server containers cannot be hosted on the same node when the `host` network configuration is used because they will all try to use the same port.